lldb/test/Shell/SymbolFile/PDB/ast-restore.test

```diff
@@ -1,11 +1,9 @@
 REQUIRES: system-windows, msvc
```

Comment:
The tests in SymbolFile/PDB should probably move to SymbolFile/NativePDB for consistency